src/main/java/com/appdirect/sdk/meteredusage/model/UsageItem.java

```diff
@@ -24,6 +24,6 @@
 	private BigDecimal unitPrice;
 	private String description;
 	private Currency currency;
-	private List<AttributeItem> attributesList;
+	private Map<String, String> attributes = null;
```

Comment:
no need for the instatiation here:
```suggestion
	private Map<String, String> attributes;
```